en/integrations/plugin-packs/procedures/applications-emc-ppma-restapi.md

```diff
@@ -0,0 +1,150 @@
+---
+id: applications-emc-ppma-restapi
+title: EMC PPMA Rest API
+---
+
+## Plugin-Pack Assets
+
+### Monitored Objects
+
+The Plugin-Pack EMC PowerPath Management Appliance collects metrics for:
+* Hosts
+
```

Comment:
- Missing host discovery part. Check: https://github.com/centreon/centreon-documentation/blob/master/en/integrations/plugin-packs/procedures/cloud-gcp-storage.md#discovery-rules

Reply:
there is no host discovery.

Reply:
we will add it later